sdk/computeschedule/arm-computeschedule/src/api/computeScheduleContext.ts

```diff
@@ -28,9 +31,10 @@ export function createComputeSchedule(
   subscriptionId: string,
   options: ComputeScheduleClientOptionalParams = {},
 ): ComputeScheduleContext {
-  const endpointUrl = options.endpoint ?? options.baseUrl ?? "https://management.azure.com";
+  const endpointUrl =
+    options.endpoint ?? getArmEndpoint(options.cloudSetting) ?? "https://management.azure.com";
   const prefixFromOptions = options?.userAgentOptions?.userAgentPrefix;
-  const userAgentInfo = `azsdk-js-arm-computeschedule/1.1.0`;
+  const userAgentInfo = `azsdk-js-arm-computeschedule/1.0.0-beta.1`;
```

Comment:
why the version is first preview?

Reply:
need to wait the new release-tool released